recipes/bob.bio.video/meta.yaml

```diff
@@ -0,0 +1,107 @@
+{% set version = "2.0.3" %}
+
+package:
+  name: bob.bio.video
+  version: {{ version }}
+
+source:
+  fn: bob.bio.video-{{ version }}.zip
+  url: https://pypi.io/packages/source/b/bob.bio.video/bob.bio.video-{{ version }}.zip
+  md5: 98a185342a93ac456a422d61dd86b772
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: python -B setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - bob.extension
+    - bob.blitz
+    - bob.core
+    - bob.io.base
+    - bob.io.image
+    - bob.io.video
+    - bob.learn.activation
+    - bob.math
+    - bob.sp
+    - bob.ip.base
+    - bob.ip.color
+    - bob.ip.draw
+    - bob.ip.gabor
+    - bob.learn.linear
+    - bob.learn.em
+    - bob.measure
+    - bob.db.base
+    - bob.db.verification.utils
+    - bob.db.verification.filelist
+    - bob.db.atnt
+    - bob.bio.base
+    - bob.bio.face
+    - bob.learn.boosting
+    - bob.ip.facedetect
+    - bob.ip.flandmark
+    - matplotlib
+    - toolchain
+
+  run:
+    - python
+    - bob.extension
+    - bob.blitz
+    - bob.core
+    - bob.io.base
+    - bob.io.image
+    - bob.io.video
+    - bob.learn.activation
+    - bob.math
+    - bob.sp
+    - bob.ip.base
+    - bob.ip.color
+    - bob.ip.draw
+    - bob.ip.gabor
+    - bob.learn.linear
+    - bob.learn.em
+    - bob.measure
+    - bob.db.base
+    - bob.db.verification.utils
+    - bob.db.verification.filelist
+    - bob.db.atnt
+    - bob.bio.base
+    - bob.bio.face
+    - bob.learn.boosting
+    - bob.ip.facedetect
+    - bob.ip.flandmark
+    - matplotlib
+
+test:
+  commands:
+    - nosetests -sv bob.bio.video
+
+  imports:
+    - bob
+    - bob.bio
+    - bob.bio.video
+    - bob.bio.video.algorithm
+    - bob.bio.video.config
+    - bob.bio.video.config.database
+    - bob.bio.video.extractor
+    - bob.bio.video.preprocessor
+    - bob.bio.video.test
+    - bob.bio.video.test.dummy
+    - bob.bio.video.utils
+
+  requires:
+    - nose
+    - bob.db.mobio
+    - bob.db.youtube
+
+about:
+  home: https://www.github.com/<YourInstitution>/<YourPackage>
```

Comment:
oops sorry it's coming from it's PyPI info: https://pypi.python.org/pypi/bob.bio.video